RunoPW/dist/metadata.json

```diff
@@ -13,9 +13,9 @@
     "tr": "Müthiş İnsanlarla Dolu İlginç Bir Yer"
   },
   "url": "runo.pw",
-  "version": "1.0.0",
-  "logo": "https://i.imgur.com/utrvCGI.png",
-  "thumbnail": "https://i.imgur.com/f7C9hGS.png",
+  "version": "2.0",
+  "logo": "https://i.imgur.com/t06m0ly.png",
+  "thumbnail": "https://i.imgur.com/zMfRQH9.png",

```

Comment:
Get a higher quality version of the logo if possible
and remove the white line at the bottom of the thumbnail